tests/integration_tests/config/simulate_prod_gamma_20_deg_North.yml

```diff
@@ -17,12 +17,13 @@ CTA_SIMPIPE:
     VIEW_CONE: "0 deg 0 deg"
     ENERGY_RANGE: "100 GeV 1 TeV"
     CORE_SCATTER: "10 500 m"
-    DATA_DIRECTORY: simtools-output
+    DATA_DIRECTORY: simtools-output/directory_for_grid_upload
```

Comment:
The data directory shouldn't be the one used for grid upload but a separate one. It should be somewhere on the tmp directory or lustre in local runs. Then the directory for grid upload should be a clean one wherever the user wants to upload from (on the grid it is in the running directory which is in the tmp directory).

Reply:
Thanks for this, cost me a bit of time.

`--pack_for_grid_register` expects now a directory name. The packed files are written into this directory. Required quite an update to the integration tests. Sorry about the many changes to be reviewed.